02-start.Rmd

```diff
@@ -35,47 +39,20 @@ the R session. Only uninstalled packages are installed.
 
 ```{r warning = FALSE, message = FALSE}
 # List of packages that we need from cran and bioc 
-cran_pkg <- c("BiocManager", "bookdown", "dplyr", "ecodist", "ggplot2", 
-              "gridExtra", "kableExtra", "knitr", "scales", "vegan", "caret",
-              "ranger", "stringr")
-bioc_pkg <- c("ANCOMBC", "ape", "DESeq2",  "DirichletMultinomial", "mia",
-              "miaViz", "microbiomeDataSets")
-
-# Gets those packages that are already installed
-cran_pkg_already_installed <- cran_pkg[ cran_pkg %in% installed.packages() ]
-bioc_pkg_already_installed <- bioc_pkg[ bioc_pkg %in% installed.packages() ]
-
-# Gets those packages that need to be installed
-cran_pkg_to_be_installed <- setdiff(cran_pkg, cran_pkg_already_installed)
-bioc_pkg_to_be_installed <- setdiff(bioc_pkg, bioc_pkg_already_installed)
-```
-
-```{r warning = FALSE, message = FALSE}
-# If there are packages that need to be installed, installs them from CRAN
-if( length(cran_pkg_to_be_installed) ) {
-   install.packages(cran_pkg_to_be_installed)
-}
-```
+packages <- c("BiocManager", "ggplot2", "pheatmap", "stringr", "igraph", "ANCOMBC", "microbiome", "httpuv", "microbiomeDataSets", "mia", "caret", "ranger")
```

Comment:
Thanks!
Could we please keep the following packages as well, they are needed at [other sections of the book](https://microbiome.github.io/course_2022_FindingPheno/) for the moment:
"miaViz", "dplyr", "gridExtra", "kableExtra", "knitr", "vegan", "ecodist"